scripts/enum/mob_mod.lua

```diff
@@ -19,7 +19,7 @@ xi.mobMod =
     HP_HEAL_CHANCE         = 9,  -- can cast cures below this HP %
     SUBLINK                = 10, -- sub link group
     LINK_RADIUS            = 11, -- link radius
-    DRAW_IN                = 12, -- 1 - player draw in, 2 - alliance draw in -- only add as a spawn mod!
+    DRAW_IN                = 12, -- bitmask with different binary options for draw-in (see DRAWIN enum for options)
```

Comment:
To make this _even more obvious_ (since this is a departure from the existing logic) I would probably name this `DRAW_IN_BITMASK` or `DRAW_IN_MASK` or `DRAW_IN_SETTINGS` etc.

Reply:
> To make this _even more obvious_ (since this is a departure from the existing logic) I would probably name this `DRAW_IN_BITMASK` or `DRAW_IN_MASK` or `DRAW_IN_SETTINGS` etc.

Yep, I have updated to DRAW_IN_BITMASK, thanks for comment.